llops_conditions/task3.js

```diff
@@ -0,0 +1,7 @@
+// Завдання 3: Генерація таблиці множення
+const number = 7
+
+for (let i = 1; i < 10; i++){
+    let summma = number * i
```

Comment:
Use const instead of let

Reply:
**summa** is not really good name, it is rather **multipliedNumber** or **result**